alacritty.yml

```diff
@@ -40,6 +40,11 @@ font:
     x: 2.0
     y: -7.0
 
+  # OS X only: use thin stroke font rendering. Thin strokes are suitable
+  # for retina displays, but for non-retina you probably want this set to
+  # false.
+  use_thin_strokes: true
```

Comment:
Just realized this was only added to this particular config. Could you also add it to `alacritty_macos.yml`?